src/components/Radio/Radio.tsx

```diff
@@ -143,7 +146,11 @@ export const Radio = React.forwardRef<HTMLInputElement, RadioProps>(
     };
 
     return (
-      <div className={clsx(classes.root, className)}>
+      <div
+        role="radio"
```

Comment:
I'm curious about having this here as well as on the `<input` on how a screen reader will behave? I can dig in a bit here later today

Reply:
I added it because it was giving an error, but I'm not sure what best practice for this would be.

Reply:
Digging in would be great!